targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP_K1/PinNames.h

```diff
@@ -310,9 +310,9 @@ typedef enum {
     LED2        = PK_6, // Orange LED
     LED3        = PK_5, // Green LED
     LED4        = PK_4,
-    LED_RED     = LED1,
-    LED_ORANGE  = LED2,
-    LED_GREEN   = LED3,
+					   
```

Comment:
Can you remove pls dead spaces?

Reply:
One more, the commit msg should contain details from the comment here, please add why you are updating the file.